client/components/tinymce/style.scss

```diff
@@ -1,9 +1,9 @@
-@import 'plugins/wplink/style';
-@import 'plugins/calypso-alert/style';
-@import 'plugins/insert-menu/style';
-@import 'plugins/wpcom-help/style';
-@import 'plugins/wpcom-charmap/style';
-@import 'plugins/contact-form/style';
+@import 'plugins/wplink/wplink';
```

Comment:
We can do that. Just wasn't sure if we wanted to treat the tinymce plugins differently since they aren't 'components'.

I guess it doesn't hurt to make the convention be "all style.scss files under client will be automatically included"